_posts/2021/2021-03-08-turn-off-the-bits-of-rails-you-dont-use.md

```diff
@@ -0,0 +1,87 @@
+---
+title: "Turn off the bits of Rails you don't use"
+description: "Reduce complexity, reduce overhead"
+layout: article
+category: ruby
+image:
+  base: "2021/turn-off-the-bits-of-rails-you-dont-use"
+  alt: "Purple lit bank of audio switches"
+  source: "https://unsplash.com/photos/n4BDkIEls78"
+  credit: "Denisse Leon"
+---
+
+The Rails Framework consists of multiple interlocking systems that work together. These systems typically begin with "Action" or "Active" e.g. Active Record, Active Support, Action Pack etc.
```

Comment:
Do we need "interlocking" and "work together"? 🤔

*The name of these systems...